packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -81,14 +82,14 @@ function hasExpectedAssetData(
 }
 
 export class QuoteRequestor {
-    private readonly _rfqtMakerEndpoints: string[];
     private readonly _schemaValidator: SchemaValidator = new SchemaValidator();
-    private readonly _warningLogger: (s: string) => void;
 
-    constructor(rfqtMakerEndpoints: string[], logger: (s: string) => void = s => logUtils.warn(s)) {
-        this._rfqtMakerEndpoints = rfqtMakerEndpoints;
-        this._warningLogger = logger;
-    }
+    constructor(
+        private readonly _rfqtAssetOfferings: RfqtMakerAssetOfferings,
+        private readonly _warningLogger: (a: any) => void = a => logUtils.warn(a),
+        private readonly _infoLogger: (a: any) => void = () => undefined,
```

Comment:
Do you think this should default to `(a) => logUtils.info(a)`?

Reply:
I think it shouldn't.

One reason is that we're outputting a JSON object in the calls to `_infoLogger`, not `stringify`ed JSON, for compatibility with Kibana.  So, if we defaulted to `logUtils.log(a)`, then the output would just say `[Object]` or whatever.  We _could_ default to `logUtils.log(JSON.stringify(a))`.

But the other reason is just that I didn't want to sully the output (of the tests, or of anything else integrating with asset-swapper) with a bunch of non-natural language (JSON) output.

I could be convinced to reconsider...

Reply:
Good point 👍